java-client/src/main/java/org/apache/pegasus/client/PegasusTable.java

```diff
@@ -2109,43 +2180,68 @@ private void handleWriteLimiterException(DefaultPromise promise, String message)
   }
 
   static class Request {
-    byte[] hashKey = null;
-    byte[] sortKey = null;
-    int sortKeyCount = 0;
+    public static final byte[] EMPTY_BYTES = "".getBytes();
+
+    public static int getLength(byte[] value) {
```

Comment:
rename `getValueLength` so that is consist with other function, or you rename all of them `getLength`